hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/config/OpenApiConfiguration.kt

```diff
@@ -75,26 +71,6 @@ class OpenApiConfiguration(buildProperties: BuildProperties) {
     .addSecurityItem(SecurityRequirement().addList("global-search-role", listOf("read")))
     .addSecurityItem(SecurityRequirement().addList("prisoner-in-prison-search-role", listOf("read")))
     .addSecurityItem(SecurityRequirement().addList("prisoner-search--prisoner--ro", listOf("read")))
-
-  @Bean
-  fun openAPICustomiser(): OpenApiCustomizer = OpenApiCustomizer {
-    it.components.schemas.forEach { (_, schema: Schema<*>) ->
-      val properties = schema.properties ?: mutableMapOf()
-      for (propertyName in properties.keys) {
-        val propertySchema = properties[propertyName]!!
-        if (propertySchema is DateTimeSchema) {
```

Comment:
not needed these days and actually stops dates being converted into `DateTime` by the OpenAPI generator